File: DAC_API/Models/DTO/Submission/SubmissionResponseDTO.cs
using System;

namespace DAC_API.Models.DTO.Submission {
    public class SubmissionResponseDTO {
        public int IdSubmission { get; set; }
        public int UserId { get; set; }
        public int ChallengeId { get; set; }
        public string PhotoUrl { get; set; }
        public string Status { get; set; }
        public DateTime? CreatedAt { get; set; }

        public string Username { get; set; }
        public string ChallengeDescription { get; set; }
        public int VoteCount { get; set; }
    }
}

File: DailyAdventureChallenge/app/src/main/java/com/example/dailyadventurechallenge/data/dto/Submission/SubmissionResponseDTO.kt
package com.example.dailyadventurechallenge.data.dto.Submission

import com.google.gson.annotations.SerializedName
import java.util.Date

data class SubmissionResponseDTO(
    @SerializedName("idSubmission")
    val idSubmission: Int,
    @SerializedName("userId")
    val userId: Int,
    @SerializedName("challengeId")
    val challengeId: Int,
    @SerializedName("photoUrl")
    val photoUrl: String?,
    @SerializedName("status")
    val status: String,
    @SerializedName("createdAt")
    val createdAt: Date,
    @SerializedName("username")
    val username: String?,
    @SerializedName("challengeDescription")
    val challengeDescription: String?,
    @SerializedName("positiveVotes")
    val positiveVotes: Int,
    @SerializedName("negativeVotes")
    val negativeVotes: Int
)
